### DISCUSSION TITLE
how i get current activeTab info in PackageResource using laravel filament 3 ?

### DISCUSSION THREAD

**Discussion by dostogirmir** [filamentphp/filament#10740] [Help] [answered]
in my PackageResource suppose i have 10 column .And this table i show column on Tab base.

my question is how i show different ,different column base on my current activeTab ?

I see filament  column have hidden option check .But if i check every column i think its slow my application . 
```
  TextColumn::make('targetUp')
                ->toggleable()
               ->hidden(fn(Page $livewire) => $livewire->activeTab !== 'IP')
                ->searchable(),
                
                ```
                
   So how i over come this scenario

**Comment by leandrocfe** [comment]
*But if i check every column i think its slow my application*
Is it happening?

**Reply by dostogirmir** [reply]
No ,but this is my question its slow my application or not .

**Reply by leandrocfe** [reply, marked as answer]
It depends, but I think you can use `hidden` `visible` for each column.

**Reply by dostogirmir** [reply]
Ok.
